### ISSUE TITLE
Left Right Arrow keys not moving between TABS using JAWS 2019 and NVD2019.2

### ISSUE THREAD

**Issue by LaurenceRLewis** [w3c/aria-practices#1209]
Example of Tabs with Manual Activation keyboard issue
https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html

CHROME 77.0.3865.90 and FIREFOX 69.0.2 IE and EDGE:

Steps to replicate

- Navigate to the Example heading using NVDA (2019) or (JAWS 2019)
- Arrow key down to the TAB widget
- NVDA announces: [tab selected] Nils Frahm Agnes Obel Joke.
- Right arrow: NVDA announces: Out of tab, tab selected
- Each Right arrow: NVDA announces a series of letters (L S Space FRAHM).
- Right Arrow then moves focus to the second tab (Agnes Obel) and announces 'Out of tab tab'
- Right arrow than announces the letters of the second tab and so on.

Down and up arrow keys move focus between tabs as expected.
Sometimes tabbing to the TAB control the key strokes work as expected.
When screen reader is not running key strokes work as expected.

Am I missing something as prior to yesterday keyboard controls worked correctly for the tab control with both JAWS and NVDA.

**Comment by ZoeBijl** [comment]
Are you using the right modifier keys? For example, with VoiceOver you’ll have to use the VO keys (default: control and option) combined with the arrow keys to get the functionality as described in the APG.

**Comment by JAWS-test** [comment]
The tabs work correctly with JAWS. Your problem is that you navigate with the arrow keys to the tabs and the screen reader does not automatically switch to forms mode. The automatic switch to form mode when reading with the arrow keys is only done for input fields. For all other elements, the mode change only occurs when navigating with the tab key or after activation (for example, with the Enter key).

**Comment by ZoeBijl** [comment]
Can confirm the tabs widget works as expected when you navigate to them with the `tab` key in either JAWS or NVDA.

**Comment by LaurenceRLewis** [comment]
Yes works tabbing to the control.
